generators/client-2/templates/src/main/webapp/_index.html

```diff
@@ -26,9 +27,9 @@
         <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
     <![endif]-->
     <page-ribbon></page-ribbon>
-    <div ui-view="navbar" ng-cloak></div>
+    <jhipster-main></jhipster-main>
```

Comment:
main component.  this is  a downgraded ng2 component.  This Will have to change once we're out of hybrid mode